src/javascripts/ng-admin/Crud/list/ListActions.js

```diff
@@ -3,7 +3,7 @@
 define(function (require) {
     'use strict';
 
-    var listActionsTemplate = require('text!./ListActions.html');
+    var listActionsTemplate = require('./ListActions.html');
```

Comment:
I specify a `html` loader for all `*.html` files in Webpack config:

``` json
{ test: /\.html$/, loader: 'html' }
```

No need to specify it manually then.